docs/tiles.html

```diff
@@ -243,552 +243,94 @@ <h2>Forms for advanced users</h2>
+                <path d="m149.9 111.42h15.216c0.27104 0 0.48922 0.21818 0.48922 0.48923v14.931" style="fill:none;stroke-linecap:round;stroke-width:4.5094;stroke:#cccccc"/>
+                <path d="m209.85 66.504 33.403 33.403" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#834bce"/>
+                <path d="m244.04 96.956-3.7387 3.7387 5.1072 1.3684z" style="fill:#834bce;stroke-width:1.2031"/>
+                <path d="m209.85 66.504-33.403 33.403" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#029807"/>
+                <path d="m175.66 96.956 3.7387 3.7387-5.1072 1.3684z" style="fill:#029807;stroke-width:1.2031"/>
+                <path d="m98.89 30.589v30.141" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#029807"/>
+                <path d="m101.53 59.203h-5.2874l2.6437 4.579z" style="fill:#029807"/>
+                <path d="m45.04 102.37 18.359 14.712" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#834bce"/>
+                <path d="m64.375 114.19 1.0171 5.1888-5.0022-1.7137z" style="fill:#834bce;stroke-width:1.2032"/>
+                <path d="m45.04 102.37-12.056 14.712" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#029807"/>
+                <path d="m31.657 114.34-0.38725 5.2732 4.7603-2.3013z" style="fill:#029807;stroke-width:1.2031"/>
+                <path d="m30.375 31.165 30.141 30.141" style="fill:none;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.1279;stroke:#834bce"/>
+                <a transform="matrix(1.2031,0,0,1.2031,-185.71,-19.869)" style="opacity:.75" xlink:href="javascript:asChecker()">
+                    <rect x="168.27" y="31.08" width="22.675" height="22.675" ry=".24461" style="fill-opacity:.5;fill:#cec511;opacity:1;stroke-linecap:round;stroke-width:1.9481;stroke:#cec511"/>
+                </a>
+                <path d="m30.375 31.165 2e-5 30.141" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#029807"/>
+                <rect x="16.735" y="47.665" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#029807"/>
+                <rect x="46.885" y="47.665" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#834bce"/>
+                <rect x="46.885" y="17.525" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#cccccc"/>
+                <rect x="49.63" y="111.02" width="27.529" height="12.134" ry=".1309" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#834bce"/>
+                <a transform="matrix(2.0094 0 0 2.0094 -525.21 -208.97)" xlink:href="javascript:asHorBricks()">
```

Comment:
This seems to activate only the border of the purple brick. Same for the object with `asVerBricks`

Reply:
Are you looking at the code I just submitted?  In the latest submission, the yellow bricks are activated.

Reply:
No, I was some 10 minutes behind and just saw you fixed it. I think the screen shots in the tutorial should also change. Then we're done. Perhaps also the colors around the form fields:
https://github.com/d-bl/GroundForge/blob/4b6573e581e48eb9bd5c2c66eb1f945aa4ce51e2/docs/tiles.html#L340-L349